test/e2e/storage/csi_volumes.go

```diff
@@ -211,26 +204,29 @@ var _ = utils.SIGDescribe("CSI Volumes [Flaky]", func() {
 	// Create one of these for each of the drivers to be tested
 	// CSI hostPath driver test
 	Describe("Sanity CSI plugin test using hostPath CSI driver", func() {
-
 		var (
-			clusterRole    *rbacv1.ClusterRole
-			serviceAccount *v1.ServiceAccount
+			serviceAccount           *v1.ServiceAccount
+			combinedClusterRoleNames []string = []string{
+				csiExternalAttacherClusterRoleName,
+				csiExternalProvisionerClusterRoleName,
+				csiDriverRegistrarClusterRoleName,
+			}
 		)
 
 		BeforeEach(func() {
 			By("deploying csi hostpath driver")
-			clusterRole = csiClusterRole(cs, config, false)
-			serviceAccount = csiServiceAccount(cs, config, false)
-			csiClusterRoleBinding(cs, config, false, serviceAccount, clusterRole)
+			csiDriverRegistrarClusterRole(cs, config, false)
```

Comment:
probably a non-issue for this PR because there's a single test in this section, but if there were multiple tests, teardown of this shared global role at the beginning of every test would be disruptive... as a follow-up, might want to change this to just create the clusterrole if needed and not worry about tearing it down afterward

similar comments about the service account binding... you'd probably want to make it unique per test (include namespace in the clusterrolebinding name)